aten/src/ATen/core/aten_interned_strings.h

```diff
@@ -657,6 +657,7 @@ _(aten, rsub) \
 _(aten, sum) \
 _(aten, sum_to_size) \
 _(aten, svd) \
+_(aten, swapdim) \
```

Comment:
names should be "swapdims" and "swapaxes" (since "axes" is plural)

Reply:
Ah right. Thanks!